warehouse/pom.xml

```diff
@@ -63,17 +63,12 @@
             <dependency>
                 <groupId>com.esotericsoftware.kryo</groupId>
                 <artifactId>kryo</artifactId>
-                <version>2.20</version>
+                <version>${version.kryo}</version>
             </dependency>
             <dependency>
                 <groupId>com.esotericsoftware.kryonet</groupId>
                 <artifactId>kryonet</artifactId>
-                <version>2.20</version>
-            </dependency>
-            <dependency>
```

Comment:
This, and the other removals below are either because the same dependencies are declared in the top-level pom's dependencyManagement section, or the dependency was never referenced in any child pom.